app/components/pluggable-drop-down.js

```diff
@@ -0,0 +1,16 @@
+import Ember from "ember";
+
+export default Ember.Component.extend({
+  displayDropDownItems: false,
+
+  actions: {
+    toggleDisplayItems() {
+      this.toggleProperty("displayDropDownItems");
+    },
+
+    selectOption(item) {
+      this.get("setSelectedValue")(item);
+      this.send("toggleDisplayItems");
```

Comment:
```suggestion
      this.set("displayDropDownItems", false);
```

in case the dropdown is closed when the action is called (you never know)